src/Github.psm1

```
@@ -313,5 +497,5 @@ function Get-LogURL {
     return $logURL
 }
 
-Export-ModuleMember -Function Invoke-GithubRequest, Add-Comment, Get-AllChangedFilesInPR, New-Issue, Close-Issue, `
+Export-ModuleMember -Function Invoke-GithubRequest, Invoke-GithubGraphQL, Invoke-GithubGraphQLParallel, Add-Comment, Get-AllChangedFilesInPR, New-Issue, Close-Issue, ` 
```

PSScriptAnalyzer (1.24.0)

[warning] This line has a backtick at the end trailed by a whitespace character. Did you mean for this to be a line continuation?

Suggested fix: Remove trailing whilespace

(PSMisleadingBacktick)